.env.example

```diff
@@ -0,0 +1,5 @@
+# The base URL to which requests are made
+REACT_APP_BOOKSTORE_URL=https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps
+
+# checkout the api documentation here https://www.notion.so/Bookstore-API-51ea269061f849118c65c0a53e88a739
+REACT_APP_BOOKSTORE_API_KEY=fBtlC62Ypw5HtvjPisLo
```

Comment:
- [ ] Great work using the `dotenv` file to load the `APIURL` and `APIKEY`. Kindly add instructions in your `README.md` file to direct those who clone your repository to set up their `.env` file with the `variables` in the `.env.example` file. The reason why I asked for this is that if you don't do this, they would not be able to persist, delete or fetch books from the API endpoint. Kindly implement this suggestion. 👍